.codex/skills/gwt-fix-issue/scripts/inspect_issue.py

```
@@ -0,0 +1,786 @@
+        print(f"Assignees: {', '.join(assignee_names)}")
+
+    author = issue.get("author", {})
+    if isinstance(author, dict) and author.get("login"):
+        print(f"Author: @{author['login']}")
+
+    print(f"URL: {issue.get('url', '')}")
+
+    # Body
+    body = issue.get("body", "")
+    if body:
+        print(f"\nBODY")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)